src/main/resources/hudson/model/ExternalJob/sidepanel.jelly

```diff
@@ -34,10 +34,8 @@ THE SOFTWARE.
 	  <l:tasks>
 	    <l:task icon="images/24x24/up.png" href="${rootURL}/" title="${%Back to Dashboard}" />
 	    <l:task icon="images/24x24/search.png" href="." title="${%Status}" />
-	    <l:isAdmin>
-	      <l:task icon="images/24x24/edit-delete.png" href="delete" title="${%Delete Job}" />
-	      <l:task icon="images/24x24/setting.png" href="configure" title="${%Configure}" />
-	    </l:isAdmin>
+	    <l:task icon="images/24x24/edit-delete.png" href="delete" title="${%Delete Job}" permission="${it.DELETE}" />
+	    <l:task icon="images/24x24/setting.png" href="configure" title="${%Configure}" permission="${it.CONFIGURE}"/>
```

Comment:
`Item.EXTENDED_READ` is actually conventional here, with some tweaks to `configure.jelly` to handle view-only users.